unused status variabel - warning

Hi,

line 158 in vl53l8cx.h
```
vl53l8cx.h:158:17: warning: unused variable 'status' [-Wunused-variable]
  158 |         uint8_t status = 0;
```
Please correct this.
Thanks.

--- a/src/vl53l8cx.h
+++ b/src/vl53l8cx.h
@@ -155,7 +155,6 @@ class VL53L8CX {
         return i != size;
       }
       if (dev_spi) {
-        uint8_t status = 0;
         uint32_t position = 0;
         uint32_t data_size = 0;
         uint16_t    temp;

